src/main/java/com/aetherteam/aether_genesis/item/accessories/CrystalBottleItem.java

```diff
@@ -0,0 +1,60 @@
+package com.aetherteam.aether_genesis.item.accessories;
+
+import com.aetherteam.aether.capability.player.AetherPlayer;
+import com.aetherteam.aether.item.accessories.AccessoryItem;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResultHolder;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.TooltipFlag;
+import net.minecraft.world.level.Level;
+import top.theillusivec4.curios.api.SlotContext;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class CrystalBottleItem extends AccessoryItem {
+    public CrystalBottleItem(Properties properties) {
+        super(properties);
+    }
+
+    public InteractionResultHolder<ItemStack> use(Level level, Player player, InteractionHand hand)  {
+        ItemStack heldItem = player.getItemInHand(hand);
+        float exp = heldItem.getOrCreateTag().getFloat("Experience");
+        if (player.isShiftKeyDown()) {
```

Comment:
If the player is holding shift but the player's experience bar is empty, this still returns as a success.